### ISSUE TITLE
Default imports don't show the correct component name

### ISSUE THREAD

**Issue by sapegin** [moroshko/react-scanner#11]
Similar to #10 but with default imports:

```js
import DesignSystemIcon from '@my/design-system/Icon'
```

The component name in the stats will be `DesignSystemIcon` instead of the correct one `Icon`.

Not sure we should detect the name automatically in this case though. Maybe taking the last part of the path and give an ability to override this behavior would be a good solution?

**Comment by moroshko** [comment]
I think trying to automagically guess the component name from the module name/path won't scale.

How about adding a `getComponentName` option?

For example:

```
{
  ...
  getComponentName: ({ imported, local, moduleName }) => {
    const parts = moduleName.split('/');
    
    return parts[parts.length - 1];
  }
}
```

In #10, I basically suggest that the default is `({ imported }) => imported` instead of the current `({ local }) => local`.

**Comment by sapegin** [comment]
Yeah, that would work I guess! 🦄

**Comment by moroshko** [comment]
`getComponentName` is available in 0.5.0.

**Comment by sapegin** [comment]
Yay, it works!

https://blog.sapegin.me/til/react/finding-the-most-used-react-components-with-react-scanner/
